Detections/AzureDevOps/AzDOAdminGroupAdditions.yaml

```diff
@@ -19,7 +19,7 @@ relevantTechniques:
 query: |
 
   let timeframe = 4h;
-  // Monitor for Project Administrator adds to *any* project
+  // Change to true to monitor for Project Administrator adds to *any* project
   let MontiorAllProjects = false;
```

Comment:
Looks like typo in variable - MonitorAllProjects

Reply:
Fixed